.expeditor/config.yml

```diff
@@ -91,6 +91,18 @@ merge_actions:
     ignore_labels:
      - "Version: Skip Bump"
      - "Expeditor: Skip All"
+    only_if_modified:
+     - etc/*
+     - habitat/*
+     - inspec-bin/*
+     - lib/*
+     - omnibus/*
+     - support/*
+     - tasks/*
+     - test/*
+     - Gemfile*
+     - "*.gemspec"
+     - LICENSE
```

Comment:
There are a lot of individual files that arguably could go here - CODE_OF_CONDUCT.md, CONTRIBUTING.md. I definitely think README.md should be versioned.

Reply:
+1 on the main meta files being versioned (e.g. CODE_OF_CONDUCT.md, CONTRIBUTING.md, README.md, etc.)